### DISCUSSION TITLE
How to verify an access token with the API

### DISCUSSION THREAD

**Discussion by Cadair** [magicbug/Cloudlog#1321] [Q&A]
I have written a [little script](https://gist.github.com/Cadair/1e0a555deac70938b099b60097acde72) which uploads WSJT QSOs to cloudlog, and I was wondering if there is a request I can make to verify the access token that the user provides before a QSO is completed and I want to upload it?

**Comment by magicbug** [comment]
I'm fairly sure the answer no unless code was added that I cannot remember about. if you don't mind reading code check /application/controllers/api.php

**Comment by AndreasK79** [comment]
@Cadair it's there it seems. "cloudlog-url"/api/auth/"api-key" seems to do the trick. Have a look in controllers/Api.php -> function auth.
